imports/ui/components/profile/profile_page.js

```diff
@@ -63,3 +73,20 @@ Template.profilePage.helpers({
     return Meteor.userId() && profile.userId === Meteor.userId();
   }
 });
+
+Template.profilePage.events({
+  'click #ban-user-from-profile': (e) => {
+    var username = FlowRouter.getParam("username");
+    var user = Meteor.users.findOne({username: username});
+    var profile = Profiles.findOne({ userId: user._id });
+    Profiles.update(profile._id, {$set: {banned: true}});
+    Bert.alert("User @" + username + " banned.", "default");
```

Comment:
So here's a question about the security of this implementation. It would seem you do not check on the server side if the user executing this has permission to ban other users, is that correct? Imagine if an attacker reads this code and inserts a #ban-user-from-profile element into the DOM. Would they be able to ban arbitrary users? I know it's pretty unlikely that people would really try this kind of crazy hack on our little site, but securing things is fun. I could try running this attack if you like, to see if it works. For science. Just let me know!